auto_dev/data/templates/protocols/test_dialogues.jinja

```diff
@@ -24,7 +24,7 @@ from packages.{{ author }}.protocols.{{ snake_name }}.tests.performatives import
 def shallow_dump(model: BaseModel) -> dict:
     """Shallow dump pydantic model."""
 
-    return {name: getattr(model, name) for name in model.__fields__}
+    return {name: getattr(model, name) for name in model.__class__.model_fields}
```

Comment:
we were getting 100's of warnings when running tests using `__fields__`, because deprecated